scripts/runtime/roxdb.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+# Opens an interactive session to the specified stackrox database, by default attempts to connect to central-db in stackrox namespace.
+#
+# Accepts an optional positional argument <db>, which specifies the
+# database to connect to, valid values are "central" and "scanner"
+#
+# Example usages:
+#   roxdb central        Connects to central-db
+#   roxdb scanner        Connects to scanner-db
+
+SCRIPT="$(python3 -c 'import os, sys; print(os.path.realpath(sys.argv[1]))' "${BASH_SOURCE[0]}")"
+source "$(dirname "$SCRIPT")/../../lib/common.sh"
+
+usage() {
+  die "Usage: $0 [central/scanner] [namespace]"
```

Comment:
Do we use this namespace parameter?

Reply:
nice catch, was adding it, then removed it it, will remove

Reply:
https://github.com/stackrox/workflow/pull/102/commits/0a75233d218ffc0f49d004b7315d9fcbb159ed72